autoflip_calculators/text_detection_calculator.cc

```diff
@@ -172,88 +159,94 @@ void TextDetectionCalculator::DecodeBoundingBoxes(const Mat& scores, const Mat&
 
             cv::Point2f offset(offsetX + cosA * x1_data[x] + sinA * x2_data[x],
                            offsetY - sinA * x1_data[x] + cosA * x2_data[x]);
-            cv::Point2f p1 = Point2f(-sinA * h, -cosA * h) + offset;
-            cv::Point2f p3 = Point2f(-cosA * w, sinA * w) + offset;
+            cv::Point2f p1 = cv::Point2f(-sinA * h, -cosA * h) + offset;
+            cv::Point2f p3 = cv::Point2f(-cosA * w, sinA * w) + offset;
 
-            cv::RotatedRect r(0.5f * (p1 + p3), Size2f(w, h), -angle * 180.0f / (float)CV_PI);
+            cv::RotatedRect r(0.5f * (p1 + p3), cv::Size2f(w, h), -angle * 180.0f / (float)CV_PI);
             detections.push_back(r);
             confidences.push_back(score);
         } // end for x
     } // end for y
 }   
 
+void TextDetectionCalculator::DetectText(const cv::Mat& frame, cv::dnn::Net detector, cv::Mat& scores, cv::Mat& geometry) {
```

Comment:
Use a pointer for any return values. So since scores and geometry are returned, use the style:
cv::Mat * scores, cv::Mat * geometry